FacebookWinFormsApp/random strategy/PostRandomStrategy.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Windows.Forms;
+using BasicFacebookFeatures.viewers;
+using FacebookWrapper.ObjectModel;
+
+namespace BasicFacebookFeatures.random_strategy
+{
+    internal class PostRandomStrategy : IRandomStrategy
+    {
+        private readonly PostViewer r_PostViewer;
+        private readonly TextBox r_TextBoxUploadDate;
+
+        public PostRandomStrategy(PostViewer i_PhotoViewer, TextBox i_TextBoxUploadDate)
+        {
+            this.r_PostViewer = i_PhotoViewer;
```

Comment:
Need to change the name of `i_PhotoViewer` to "i_PostViewer"

Reply:
i already fixed thoes issues but for some reason forgot to index them in the commit.. :)